### DISCUSSION TITLE
ICamundaClientBuilder DI example

### DISCUSSION THREAD

**Discussion by nadman67** [camunda-community-hub/zeebe-client-csharp#628] [General]
There is a dependency injection example that uses IZeebeClientBuilder, but I'm having an issue using the ICamundaCloudClientBuilder. I'm getting a 'No service for type 'Zeebe.Client.Api.Builder.ICamundaCloudClientBuilder' has been registered.

**Comment by ChrisKujawa** [comment]
I guess we need to extend https://github.com/camunda-community-hub/zeebe-client-csharp/blob/main/Client/ServiceCollectionExtensions.cs
